tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -3688,6 +3689,24 @@ MklLayoutRewritePass::CheckForQuantizedNodeRewrite(const Node* n) const {
   if (type_attrs_present) {
     for (auto ri = rinfo_.cbegin(); ri != rinfo_.cend(); ++ri) {
       if (n->type_string().compare(ri->name) == 0 && ri->rewrite_rule(n)) {
+	// Currently OneDNN optimization does not support int8 with native
+	// format.
+        if (NativeFormatEnabled()) {
+          static absl::once_flag once;
+	  absl::call_once(once, [] {
+#if defined(ENABLE_MKL)
+            VLOG(0) << "MklLayoutRewritePass::RewriteInfo: Do not support INT8"
```

Comment:
Nit: This makes it clearer that the subject is `MklLayoutRewritePass::RewriteInfo`. Same for the other message.
```suggestion
            VLOG(0) << "MklLayoutRewritePass::RewriteInfo does not support INT8"
```

Reply:
changed